### DISCUSSION TITLE
Adding Categories

### DISCUSSION THREAD

**Discussion by wkuilder** [nextcloud/cookbook#1284] [FAQ and user questions] [answered]
Hi, I have just installed cookbook but can't find a way to add categories in next cloud and advise will be appreciated.

Thanks

**Comment by seyfeb** [comment, marked as answer]
Hi wkbuilder, just create a new recipe or edit an existing recipe. In the category field, you can simply type in a new category name, hit enter to confirm, and save the recipe and the category will be available from the list of categories.
